aerospike_enterprise/metadata.csv

```diff
@@ -0,0 +1,651 @@
+aerospike.namespace.mrt_verify_read_timeout,gauge,,,,Number of verify read commands that timed out.,0,aerospike,,,
+aerospike.namespace.nodes_quiesced,gauge,,node,, The number of nodes observed to be quiesced as of the most recent reclustering event.,0,aerospike,,,
+aerospike.namespace.non_expirable_objects,gauge,,,,The number of records in this namespace with non-expirable TTLs (TTLs of value 0).,0,aerospike,,,
+aerospike.namespace.non_replica_objects,gauge,,record,,The number of records on this node which are neither master nor replicas,0,aerospike,,,
+aerospike.namespace.non_replica_tombstones,gauge,,,,The number of tombstones on this node which are neither master nor replicas. This number is non-zero only during migration.,0,aerospike,,,
+aerospike.namespace.ns_cluster_size,gauge,,,,The namespace cluster size,0,aerospike,,,
+aerospike.namespace.ns_forward_xdr_writes,gauge,,,,This parameter provides fine grained control at namespace level to forward writes that originated from another XDR to the specified destination datacenters (in xdr section).,0,aerospike,,,
+aerospike.namespace.nsup_cycle_deleted_pct,gauge,,,,Percent of records removed by NSUP in its last cycle.,0,aerospike,,,
+aerospike.namespace.nsup_cycle_duration,gauge,,millisecond,,The length of the last nsup cycle.,0,aerospike,,,
+aerospike.namespace.nsup_hist_period,gauge,,,,"The interval (secs) at which the object size histograms, as well as the time-to-live (ttl) histogram, are updated.",0,aerospike,,,
+aerospike.namespace.nsup_period,gauge,,,,"The interval at which the main expiration/eviction thread (called nsup, the namespace supervisor) wakes up to process the namespace.",0,aerospike,,,
+aerospike.namespace.nsup_threads,gauge,,thread,,The number of dedicated expiration/eviction threads for nsup to use when processing the namespace.,0,aerospike,,,
+aerospike.namespace.nsup_xdr_key_busy,gauge,,,,Number of NSUP deletes (expirations and evictions) that had to wait for a previous version to ship,0,aerospike,,,
+aerospike.namespace.objects,gauge,,record,,The number of records in this namespace for this node. Does not include tombstones.,0,aerospike,Namespace Objects,,
+aerospike.namespace.ops_sub_tsvc_error,gauge,,transaction,,The number of ops scan/query sub-transactions that failed with an error in the transaction service.,0,aerospike,,,
+aerospike.namespace.ops_sub_tsvc_timeout,gauge,,transaction,,The number of ops scan/query sub-transactions that timed out in the transaction service.,0,aerospike,,,
+aerospike.namespace.ops_sub_write_error,gauge,,transaction,,The number of ops scan/query write sub-transactions that failed with an error. Does not include timeouts.,0,aerospike,,,
+aerospike.namespace.ops_sub_write_filtered_out,gauge,,transaction,,The number of ops scan/query write sub-transactions for which the write did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
```

Comment:
```suggestion
aerospike.namespace.ops_sub_write_filtered_out,gauge,,transaction,,The number of ops scan/query write sub-transactions for which the write did not happen because the record was filtered out using a predicate expression.,0,aerospike,,,
```